setup.py

```diff
@@ -60,6 +60,7 @@
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     install_requires=[],
     extras_require={
+        'pyproject': ['toml'],
         'test': ['mock'],
```

Comment:
Suggestion: add toml to the [test] extra, because the tests rely on it being installed.